src/main/resources/openshift/deploy.sh

```diff
@@ -39,18 +39,32 @@ rc=$?; if [[ $rc != 0 ]]; then echo "Missing deployment. Please build and copy r
 echo
 echo "Openshift project"
 echo "  -> Create Openshift project (${OCP_PROJECT})"
-oc new-project ${OCP_PROJECT} 2>/dev/null > /dev/null
+oc new-project ${OCP_PROJECT} > /dev/null
 sleep 1
-
-oc policy add-role-to-user view system:serviceaccount:$(oc project -q):eap-service-account -n $(oc project -q)
-oc policy add-role-to-user view system:serviceaccount:$(oc project -q):sso-service-account -n $(oc project -q)
-
 echo "  -> Switch to project"
-oc project ${OCP_PROJECT}  2>&1 > /dev/null
+oc project ${OCP_PROJECT} > /dev/null
+sleep 1
+echo "  -> Register service accounts"
```

Comment:
Techically this isn't registering service accounts. It is giving them sufficient privileges to support accessing the Kubernetes APIs, which prevents warnings later on.

"Register" might be easier for the user to understand, though.